public/javascripts/overlays/admin.js

```diff
@@ -607,9 +607,8 @@ var challongeUrl;
 challongeSocket.on('update challonge', function(data){
```

Comment:
Should this namespace and its associates be renamed to be agnostic? `bracketSocket` or something.